libs/langgraph/langgraph/pregel/algo.py

```diff
@@ -452,6 +452,8 @@ def prepare_next_tasks(
                 triggered_nodes.update(node_ids)
         # Sort the nodes to ensure deterministic order
         candidate_nodes: Iterable[str] = sorted(triggered_nodes)
+    elif not checkpoint["channel_versions"]:
```

Comment:
an inline doc would be helpful here -- it's not obvious what this means semantically -- requires knowledge of how / when channel_versions in a checkpoint are set

I can update after the fact fwiw